recipes/libstudxml/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, MSBuild, tools
+import os
+
+class LibStudXmlConan(ConanFile):
+    name = "libstudxml"
+    description = "A streaming XML pull parser and streaming XML serializer implementation for modern, standard C++."
+    topics = ("xml", "xml-parser", "serialization")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.codesynthesis.com/projects/libstudxml/"
+    license = "MIT"
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("expat/2.4.1")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def _configure_autotools(self):
+        if not self._autotools:
+            args = ["--with-extern-expat", "CXXFLAGS=\"-UNDEBUG\""]
+            if self.options.shared:
+                args.append("--disable-static")
+            else:
+                args.append("--disable-shared")
```

Comment:
Better to be specific
```suggestion
            if self.options.shared:
                args.extend(["--enable-shared", "--disable-static"])
            else:
                args.extend(["--disable-shared", "--enable-static"])
```